ports/atmel-samd/boards/picoplanet/pins.c

```diff
@@ -0,0 +1,36 @@
+#include "shared-bindings/board/__init__.h"
+
+STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
+	
+
+	{ MP_ROM_QSTR(MP_QSTR_A1), MP_ROM_PTR(&pin_PA02) },
+	{ MP_ROM_QSTR(MP_QSTR_A0), MP_ROM_PTR(&pin_PA03) },
+	{ MP_ROM_QSTR(MP_QSTR_A2), MP_ROM_PTR(&pin_PA04) },
+	
+
+	{ MP_ROM_QSTR(MP_QSTR_D5),MP_ROM_PTR(&pin_PA05) },
+	{ MP_ROM_QSTR(MP_QSTR_D6),MP_ROM_PTR(&pin_PA06) },
+	{ MP_ROM_QSTR(MP_QSTR_D7),MP_ROM_PTR(&pin_PA07) },
+	
+
+	{ MP_ROM_QSTR(MP_QSTR_D1), MP_ROM_PTR(&pin_PA08) },
+    { MP_ROM_QSTR(MP_QSTR_A3), MP_ROM_PTR(&pin_PA08) },
+    { MP_ROM_QSTR(MP_QSTR_SDA), MP_ROM_PTR(&pin_PA08) },
+    
+    { MP_ROM_QSTR(MP_QSTR_D2), MP_ROM_PTR(&pin_PA09) },
+    { MP_ROM_QSTR(MP_QSTR_A4), MP_ROM_PTR(&pin_PA09) },
+    { MP_ROM_QSTR(MP_QSTR_SCL), MP_ROM_PTR(&pin_PA09) },
+    
+    { MP_ROM_QSTR(MP_QSTR_D3), MP_ROM_PTR(&pin_PA16) },
+    { MP_ROM_QSTR(MP_QSTR_MOSI), MP_ROM_PTR(&pin_PA16) },
+    
+    { MP_ROM_QSTR(MP_QSTR_D4), MP_ROM_PTR(&pin_PA17) },
+    { MP_ROM_QSTR(MP_QSTR_SCK), MP_ROM_PTR(&pin_PA17) },
+    
+    { MP_ROM_QSTR(MP_QSTR_D8), MP_ROM_PTR(&pin_PA30) },
+    { MP_ROM_QSTR(MP_QSTR_MISO), MP_ROM_PTR(&pin_PA30) },
+	
+    { MP_ROM_QSTR(MP_QSTR_I2C), MP_ROM_PTR(&board_i2c_obj) },
+    { MP_ROM_QSTR(MP_QSTR_SPI), MP_ROM_PTR(&board_spi_obj) }
```

Comment:
As long as you're editing this, could you touch up the indentation here, and remove the double blank lines?

Reply:
Hi, thanks so much for your feedback! I fixed indentation and double blank lines, but it seems pre-commit still fails. Did I miss something?